### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by tieugene** [ArsMasiuk/qvge#94]
```
src/coin/CoinUtils/CoinMessageHandler.cpp:766:35: error: format not a string literal and no format arguments [-Werror=format-security]
  766 |      sprintf(messageOut_,format_+2);
      |                                   ^
cc1plus: some warnings being treated as errors
```
gcc-c++ 10.1.1, Qt 5.14.2

**Comment by ArsMasiuk** [comment]
This is the problem with 3rd party library OGDF (#83) which should be replaced in the future.

**Comment by ArsMasiuk** [comment]
Should be fixed for 0.6
